src/OpenTelemetry.Extensions/README.md

```diff
@@ -37,3 +37,33 @@ public static TracerProviderBuilder AddMyExporter(this TracerProviderBuilder bui
         .AddAutoFlushActivityProcessor(a => a.Parent == null && (a.Kind == ActivityKind.Server || a.Kind == ActivityKind.Consumer), 5000);
 }
 ```
+
+### BaggageSpanProcessor
```

Comment:
Given .NET already settled on `Activity`, we should name this `BaggageActivityProcessor`, like `AutoFlushActivityProcessor` already part of this package.